horovod/spark/lightning/remote.py

```diff
@@ -101,7 +108,11 @@ def train(serialized_model):
 
             # Use default logger if no logger is supplied
             train_logger = logger
-            print(f"Train_logger is {train_logger}")
+
+            # Resume logger experiment key if passed from CPU.
+            if hasattr(train_logger, '_experiment_key') and train_logger._experiment_key is None:
```

Comment:
Is this `CometLogger` specific?
Then we can use `isinstanceof(CometLogger)`, since `CometLogger` is imported on line#27